Makefile

```diff
@@ -0,0 +1,2 @@
+test:
+	pytest -v liger_iris_pipeline/tests/test_datamodels.py liger_iris_pipeline/tests/test_dark.py liger_iris_pipeline/tests/test_rop.py liger_iris_pipeline/tests/test_utils.py liger_iris_pipeline/tests/test_normalize.py liger_iris_pipeline/tests/test_assign_wcs.py liger_iris_pipeline/tests/test_asdf_schema.py liger_iris_pipeline/tests/test_preprocess_flatfield.py
```

Comment:
need to add `image2` to the makefile

Reply:
Removed in [8f84054](https://github.com/oirlab/liger_iris_pipeline/pull/75/commits/8f84054f3e1f4a5a62d17bcc61e73e917c49220f)